scripts/paper/generate_stwo_phase12_shared_lookup_bundle_figure.py

```
@@ -0,0 +1,358 @@
+  {left_panel}
+  {right_panel}
+  {render_text(95, 830, footnote_lines[0], size=17, anchor="start", fill="#6B7280")}
+  {render_text(95, 858, footnote_lines[1], size=17, anchor="start", fill="#6B7280")}
+</svg>
+'''
+
+
+def write_optional_rasters(svg_path: Path, png_path: Path, pdf_path: Path, *, fail_closed: bool) -> None:
+    tmp_png = png_path.with_suffix(png_path.suffix + ".tmp")
+    try:
+        rsvg = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)